packages/functional-tests/src/app.ts

```diff
@@ -85,7 +87,14 @@ export default class App {
             const test = this.activeTests[testName] = this.testFactories[testName](user);
             this.rpc.send('functional-test:test-started', testName);
             console.log(`Test started: '${testName}'`);
-            const success = await test.run();
+            let success: boolean;
+            try {
+                success = await test.run();
+            }
+            catch (e) {

```

Comment:
catch should appear on same line as `}` --> `} catch (e) {`. I think tslint isn't working for you in this project.